frontend/src/pages/index.js

```diff
@@ -1,5 +1,6 @@
 import React from 'react';
 import Layout from '../components/layout';
+import List from '../components/aeppslist';
```

Comment:
Change file name to **aepps-list.js**
`import AeppList from '../components/aepps-list';`